pay-api/src/pay_api/models/eft_short_names.py

```diff
@@ -78,9 +71,9 @@ class EFTShortnameSchema:  # pylint: disable=too-few-public-methods
     transaction_date: datetime
     deposit_date: datetime
     deposit_amount: Decimal
-    linked_by: str
-    linked_by_name: str
-    linked_on: datetime
+    updated_by: str
```

Comment:
Renamed to be more generic due to the different statuses possible now. The status will indicate the action,

LINKED = linked by, etc

link history table will track any transitions and who did it